site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/converter/OrganizationConverter.java

```diff
@@ -7,7 +7,6 @@
 import uk.ac.ox.ndph.mts.site_service.model.Site;
 
 /**
- * Implement an EntityConverter for Site
  */
```

Comment:
[nit] If you deleted the comment text, can you also remove the actual comment pre/suffix?

Reply:
Done